### PR TITLE
Update install.rst

### PR DESCRIPTION
**Pull request by Jac-eko**
Add apt command to install GNU-RM toolchain
I'm not sure if there is too much packages, but it works

### CODE REVIEW

**Comment by panos** [comment]
> Add apt command to install GNU-RM toolchain
> I'm not sure if there is too much packages, but it works

I think that could confuse people using other operating systems, unless if custom commands were to be written for as many case scenarios as possible.

**Comment by daniel-thompson** [comment]
Thanks for the PR although I confess I am a little reluctant to accept it.

The cross-compiler toolsets included with the distros are not the same as the Arm GNU-RM toolchain. The reason I explicitly indicate that all build and test is done using the Arm GNU-RM toolchain is that in the past there have been some significant problems with arm-none-eabi toolchains on some distros, espeically when building floating point code.

In other words if the Debian Buster toolchain works then great but it's not what we normally build and test with and, for that reason, is not the recommended toolchain.
